h2o-cve-2018-0608/README.md

```diff
@@ -0,0 +1,43 @@
+# h2o memory corruption CVE example
+
+This repo replicates finding a memory corruption bug inside h2o with fuzzing.
```

Comment:
Please expand this.  For example:
This repo replicates finding [CVE-2018-0608](https://nvd.nist.gov/vuln/detail/CVE-2018-0608), a memory corruption bug that may allow a remote attacker to run arbitrary code (CVSS Score: 9.8).

We reported this bug responsibly to the maintainers, with the follow-on issue tracking [here](https://github.com/h2o/h2o/issues/1775).

Reply:
I'm going to steal this :)